src/utils/post.js

```diff
@@ -0,0 +1,27 @@
+async function postData(user, score) {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/rO378WlzTyLm1i3wuLMC/scores/', {
+    method: 'POST',
+    body: JSON.stringify({
+      user,
+      score,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  const json = await response.json();
+  return json;
+}
+
+export default function recieveData() {
+  const msg = document.querySelector('.msg');
```

Comment:
- [ ] Kindly make use of `arrow functions` instead of the `function` keyword to follow ES6 best practices.